test/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/BlazorLegacyIntegrationTest60.cs

```diff
@@ -6,8 +6,7 @@
 
 namespace Microsoft.NET.Sdk.BlazorWebAssembly.Tests
 {
-    public class BlazorLegacyIntegrationTest60(ITestOutputHelper log)
-        : IsolatedNuGetPackageFolderAspNetSdkBaselineTest(log, nameof(BlazorLegacyIntegrationTest60))
+    public class BlazorLegacyIntegrationTest60(ITestOutputHelper log) : BlazorWasmBaselineTests(log, GenerateBaselines)
```

Comment:
This test needs nuget package isolation, otherwise it interferes with other tests.

We would require something like IsolatedNuGetPackageFolderBaselineTest

Reply:
I made some changes, but it looks like this is not the real reason for the test failure.